### PR TITLE
Decor page layout and styles 

### PR DESCRIPTION
**Pull request by halynarydasova**

### CODE REVIEW

File: frontend/src/pages/Decor/Decor.module.scss [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    transform: rotate(3.92deg);
+  }
+  &:nth-child(3) {
+    width: 440px;
+    height: 319px;
+    right: 0;
+    top: 338px;
+  }
+}
+.cover {
```
Comment (Svitlana-G) [inline frontend/src/pages/Decor/Decor.module.scss:79]: I didn't find className `cover i`n Decor.jsx

File: frontend/src/pages/Decor/Decor.module.scss [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(255, 255, 255, 0.4);
+  z-index: 1;
+}
+.successPopup {
```
Comment (Svitlana-G) [inline frontend/src/pages/Decor/Decor.module.scss:88]: I didn't find className `successPopup `in Decor.jsx

File: frontend/src/pages/Decor/Decor.jsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import styles from './Decor.module.scss';
+import Path from '../CardPage/components/Path';
+import { useForm } from 'react-hook-form';
+import { IMaskInput } from 'react-imask';
+import Button from '../../components/ui-kit/components/Button';
+import axiosInstance from '../../services/httpClient';
+import { useNavigate } from 'react-router-dom';
+
+const addDecorRequest = async (request) => {
```
[the remark this answers is not in the document]
Comment (halynarydasova) [reply frontend/src/pages/Decor/Decor.jsx:10]: done

File: frontend/src/pages/Decor/Decor.jsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  try {
+    const response = await axiosInstance.post('/decor', request);
+    if (response.status === 200) {
+      return response.data.id;
```
[the remark this answers is not in the document]
Comment (halynarydasova) [reply frontend/src/pages/Decor/Decor.jsx:14]: Yup, i use it further to redirect user to main page, if the request was successful. I changed it to status though

File: frontend/src/pages/Decor/Decor.module.scss [hunk 5]
```diff
@@ -0,0 +1,172 @@
+@import '/src/variables';
+@import '/src/mixins';
+@mixin flower {
```
[the remark this answers is not in the document]
Comment (halynarydasova) [reply frontend/src/pages/Decor/Decor.module.scss:3]: Make sense

File: frontend/src/pages/Decor/Decor.module.scss (hunk 1 above)
Comment (halynarydasova) [reply frontend/src/pages/Decor/Decor.module.scss:79]: I removed it

File: frontend/src/pages/Decor/Decor.module.scss (hunk 2 above)
Comment (halynarydasova) [reply frontend/src/pages/Decor/Decor.module.scss:88]: Removed. Done

**Pull request merged**